google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BlobInfo.java

```diff
@@ -423,12 +441,37 @@ public Builder setMd5(String md5) {
       return this;
     }
 
+    public Builder setMd5FromHexString(String md5HexString) {
+      if (md5HexString == null) {
+        return this;
+      }
+      if (md5HexString.startsWith("0x")) {
+        md5HexString = md5HexString.replaceFirst("0x", "");
+      }
+      byte[] bytes = new BigInteger(md5HexString, 16).toByteArray();
+      this.md5 = "0x" + BaseEncoding.base64().omitPadding().encode(bytes);
```

Comment:
Please remove prepending "0x".

Reply:
@frankyn, do you mean the change for both Md5 and Crc32c, correct 
What about unit tests values? https://github.com/googleapis/google-cloud-java/blob/7de46b4ce194f4c7d7094dd1d7dec4f4a85ad363/google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BlobInfoTest.java#L55

We allow user to enter the value with 0x and without.